plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -5505,13 +5505,15 @@ img.help_tip {
   * WooCommerce meta boxes
   */
 .wc-metaboxes-wrapper {
+	&:not(#variable_product_options_inner) {
```

Comment:
Haven't spent much time looking into what the selector should be, but this isn't right. 

`#variable_product_options_inner` is a child of `.wc-metaboxes-wrapper`